camel/toolkits/arxiv_toolkit.py

```diff
@@ -98,10 +100,22 @@ def search_papers(
                 "authors": [author.name for author in paper.authors],
                 "entry_id": paper.entry_id,
                 "summary": paper.summary,
-                # TODO: Use chunkr instead of atxiv_to_text for better
-                # performance
-                "paper_text": arxiv_to_text(paper.pdf_url),
+                "pdf_url": paper.pdf_url,
             }
+
+            # Extract text from the paper if the PDF URL is valid
+            text = ""
+            if check_url_validity(paper_info["pdf_url"]):
+                try:
+                    # TODO: Use chunkr instead of atxiv_to_text for better
+                    # performance and reliability
+                    text = arxiv_to_text(paper_info["pdf_url"])
+                except Exception:
+                    pass
+            else:
+                pass
```

Comment:
we shouldn't just use `pass` without any handling for `except` in our framework code, at least we need to do 
```
        except Exception as e:
            return f"An error occurred: {e}"
```

Reply:
Thanks, I changed to except handler to set `text = ""`.